requirements_dev.txt

```diff
@@ -12,3 +12,5 @@ ptvsd==4.3.2
 freezegun==0.3.12
 watchdog==0.9.0
 ptpython==3.0.17
+nzpy>=1.15
+nzalchemy
```

Comment:
`requirements_dev.txt` is only for development dependencies, needed for unit tests and other general development tasks. See earlier comment about where to specify query runner dependencies.

```suggestion
```